tests/Presenters/ItemSubdomainPresenter.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Coderflex\LaravelPresenter\Tests\Presenters;
+
+;
+
+use Coderflex\LaravelPresenter\Presenter;
+
+class ItemSubdomainPresenter extends Presenter
+{
+    public function caps()
+    {
+        return str($this->model->title)->upper()->toString();
```

Comment:
Same here, use the `Str` facade, or invoke the `str` function on the namespaces.